### PR TITLE
[terra-outline-view-table] Rename outline-table-view to outline-view-table

### PR DESCRIPTION
**Pull request by neilpfeiffer**
### Summary

Corrects accidental initial misnamed package to OutlineView-base naming convention (OutlineViewTable, OutlineViewList, etc)
- Changes all "outline table view" to "outline view" + "table"

### Deployment Link

- https://terra-core-rename-outli-xvsvja.herokuapp.com/components/terra-outline-view-table/outline-view-table/about
- https://terra-core-rename-outli-xvsvja.herokuapp.com/tests/terra-outline-view-table/outline-view-table/outline-view-table

### CODE REVIEW

File: packages/terra-outline-view-table/tests/jest/Table.test.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import OutlineTableView from '../../src/Table';
+import OutlineTableView from '../../src/OutlineViewTable';
```
Comment (sycombs) [inline packages/terra-outline-view-table/tests/jest/Table.test.jsx:3]: Should this also be `OutlineViewTable`? Same for [here](https://github.com/cerner/terra-core/blob/aa978cd60fd0dfabac3047e2adf844b41eaf12ae/packages/terra-outline-view-table/src/terra-dev-site/doc/outline-view-table/example/DefaultTable.jsx#L2) and [here](https://github.com/cerner/terra-core/blob/aa978cd60fd0dfabac3047e2adf844b41eaf12ae/packages/terra-outline-view-table/src/terra-dev-site/test/outline-view-table/OutlineViewTable.test.jsx#L2).
```suggestion
import OutlineViewTable from '../../src/OutlineViewTable';
```

File: packages/terra-outline-view-table/src/terra-dev-site/test/outline-view-table/OutlineViewTable.test.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import OutlineTableView from 'terra-outline-table-view';
+import OutlineTableView from 'terra-outline-view-table';
 
 const DefaultTest = () => (
   <OutlineTableView
```
Comment (nikhitasharma) [inline packages/terra-outline-view-table/src/terra-dev-site/test/outline-view-table/OutlineViewTable.test.jsx:5]: Should the changes be done here as well

File: packages/terra-outline-view-table/tests/jest/Table.test.jsx (hunk 1 above)
Comment (neilpfeiffer) [reply packages/terra-outline-view-table/tests/jest/Table.test.jsx:3]: Good catch. I'll update.

File: packages/terra-outline-view-table/src/terra-dev-site/test/outline-view-table/OutlineViewTable.test.jsx (hunk 2 above)
Comment (neilpfeiffer) [reply packages/terra-outline-view-table/src/terra-dev-site/test/outline-view-table/OutlineViewTable.test.jsx:5]: Good catch. I'll update.

File: packages/terra-outline-view-table/tests/jest/Table.test.jsx (hunk 1 above)
Comment (neilpfeiffer) [reply packages/terra-outline-view-table/tests/jest/Table.test.jsx]: https://github.com/cerner/terra-core/pull/3692/commits/5d5e261a193d64dec07f1f49f29f8e2ee8baf2c7

File: packages/terra-outline-view-table/src/terra-dev-site/test/outline-view-table/OutlineViewTable.test.jsx (hunk 2 above)
Comment (neilpfeiffer) [reply packages/terra-outline-view-table/src/terra-dev-site/test/outline-view-table/OutlineViewTable.test.jsx]: https://github.com/cerner/terra-core/pull/3692/commits/5d5e261a193d64dec07f1f49f29f8e2ee8baf2c7

File: packages/terra-outline-view-table/src/terra-dev-site/doc/outline-view-table/example/DefaultTable.jsx [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import OutlineTableView from 'terra-outline-table-view';
+import OutlineViewTable from 'terra-outline-table-view';
```
Comment (sycombs) [inline packages/terra-outline-view-table/src/terra-dev-site/doc/outline-view-table/example/DefaultTable.jsx:2]: Should this be changed as well?
```suggestion
import OutlineViewTable from 'terra-outline-view-table';
```

Comment (neilpfeiffer) [reply packages/terra-outline-view-table/src/terra-dev-site/doc/outline-view-table/example/DefaultTable.jsx]: Updated https://github.com/cerner/terra-core/pull/3692/commits/ba8ab299b27ddc4eb14861ac8624ed6e197bcb16

**Review by nikhitasharma — APPROVED** [review]

**Pull request merged**